mlflow/cli.py

```diff
@@ -243,19 +243,23 @@ def _user_args_to_dict(arguments, argument_type="P"):
     return user_dict
 
 
-def _validate_server_args(gunicorn_opts=None, workers=None, waitress_opts=None):
```

Comment:
Start using uvicorn + fastapi in order to support higher throughput span logging (see https://github.com/mlflow/mlflow/pull/17039/files#r2252280972 for context).

I filed a separate PR for this already: https://github.com/mlflow/mlflow/pull/17038